### DISCUSSION TITLE
Update via OTA to firmware 97bc964 does not change WEB-UI

### DISCUSSION THREAD

**Discussion by huckes** [tbnobody/OpenDTU#627] [Q&A]
How do I manage to get the Web-UI updated ?

Hi, I just updated an already running OpenDTU with the latest version 97bc964. With this version also some typos in the UI were corrected and in the NTP-settings the location configuration was added. But these and other changes are not visible after OTA-update, although the menu Info - System shows the firmware as 'Up to date!'.

**Comment by tbnobody** [comment]
Press F5 or Ctrl+F5. Depending on your browser

**Comment by huckes** [comment]
That works. Thank's a bunch. :-)
